### ISSUE TITLE
Runtime exception with dependencies

### ISSUE THREAD

**Issue by mehdichaouch** [AOEpeople/MageTestStand#18]
Hello,

When running the install as usual:
```
export WORKSPACE=/home/Oyst_OneClick-1.0.0-dev-27
export MAGENTO_VERSION=magento-mirror-1.9.3.1

export MAGENTO_DB_HOST=localhost
export MAGENTO_DB_PORT=22
export MAGENTO_DB_USER=root
export MAGENTO_DB_PASS=123
#export MAGENTO_DB_NAME=somename

curl -sSL https://raw.githubusercontent.com/AOEpeople/MageTestStand/master/setup.sh | bash
```

I get the error:
![image](https://user-images.githubusercontent.com/861701/30870625-aae2e878-a2e4-11e7-88d0-9684c29c605f.png)

`
  [RuntimeException]
  Could not load package shyim/shopware-profiler in http://packagist.org: [UnexpectedValueException] Could not parse version constraint ^v1.2.17: Invalid version string "^v1.2.17"
`

You can see the error in Travis also: https://travis-ci.com/oystparis/oyst-1click-magento/builds/55206763#L658

Do you have any idea? or solution?

**Comment by sprankhub** [comment]
I have the same issue right now.

**Comment by sprankhub** [comment]
Should be fixed with the composer update in 9b0e038c849ddaad225770b48bff182bfb8f7c1a

**Comment by mehdichaouch** [comment]
I confirm that the composer update fixed the problem, my build is 
![image](https://user-images.githubusercontent.com/861701/31271956-7f95283c-aa89-11e7-9823-d69e1a53dba4.png)

Thank you 🍾
